openapi2kong/validator.go

```diff
@@ -81,6 +81,12 @@ func generateParameterSchema(operation *v3.Operation, path *v3.PathItem,
 			paramConf["style"] = style
 			paramConf["explode"] = explode
 			paramConf["in"] = parameter.In
+
+			if parameter.In == "cookie" {
+				return nil, fmt.Errorf(`cookie parameters are not supported for request-validator plugin; 
+				choose either path, query or header`)
```

Comment:
I don't think it should throw an error, but should log a warning, and continue without generating the validation logic.

Similar to bodies, where the content type isn't json.

Reply:
and most likely we should make it more explicit in the docs, that Cookie isn't supported